packages/block/README.md

```diff
@@ -9,6 +9,15 @@
 | Implements schema and functions related to Ethereum blocks. |
 | ----------------------------------------------------------- |
 
+- 🦄 All block features till **Pectra**
+- 🌴 Tree-shakeable API
+- 👷🏼 Controlled dependency set (4 external + `@Noble` crypto)
+- 🔮 `EIP-4844` Shard Blob Txs
+- 💸 `EIP-4895` Beacon Chain Withdrawals
+- 📨 `EIP-7685` Consensus Layer Requests
+- 🛵 324KB bundle size (81KB gzipped)
```

Comment:
Ok, this is fine, but for every PR which we will merge in any of these packages these numbers will likely be incorrect. (Just an informational note. Should we update it for each release to reflect the bundle size?)

Reply:
I think we could also consider not including the size here in the docs. There are bundle size tools for this (https://bundlephobia.com/package/@ethereumjs/block@5.3.0)

Reply:
I think it is super important to keep and brings great additional value if we do this (Paul is doing as well). This does not need to be "on point" by KB but gives users strong guidance if they can (bundle size is ok for them, just ballpark number) or cannot (bundle size far too high) use our libraries.